utils_edx_cookiecutters/run_make.py

```diff
@@ -0,0 +1,12 @@
+"""
+Utility tool to run make targets in current directory
+"""
+
+from test_utils.venv import run_in_virtualenv
```

Comment:
Should probably lift that module into the main source directory if it's going to be called from source modules. Maybe `utils_edx_cookiecutters/internal/venv.py`.